index.css

```diff
@@ -112,14 +115,9 @@ body p {
   font-weight: 600;
```

Comment:
Why don't you just set the font-weight on your global h2 style and omit this here?